apps/test/unit/lib/ui/accounts/ManageLinkedAccountsTest.jsx

```diff
@@ -76,24 +75,26 @@ describe('ManageLinkedAccounts', () => {
     expect(googleEmailCell).to.have.text('teacher@email.com');
   });
 
-  it('calls connect if authentication option is not connected', () => {
-    const connect = sinon.stub();
+  it('navigates to provider endpoint if authentication option is not connected', () => {
     const wrapper = mount(
       <ManageLinkedAccounts
         {...DEFAULT_PROPS}
-        connect={connect}
       />
     );
+    const navigateToHrefStub = sinon.stub(utils, 'navigateToHref');
     wrapper.find('BootstrapButton').at(0).simulate('click');
-    expect(connect).to.have.been.calledOnce;
+    let arg = navigateToHrefStub.getCall(0).args[0];
+    expect(navigateToHrefStub).to.have.been.calledOnce;
+    expect(arg).to.equal('/users/auth/google_oauth2/connect');
```

Comment:
A couple nits in how this is expressed:
```js
expect(utils.navigateToHref).to.have.been.calledOnce
  .and.calledWith('/users/auth/google_oauth2/connect');
```

I have a slight preference for referring to the stubbed method itself (feel free to push back). See [`sinon-chai`](https://github.com/domenic/sinon-chai) for the `calledWith` assertion.